pkg/ccl/backupccl/restore_mid_schema_change_test.go

```diff
@@ -221,7 +221,12 @@ func restoreMidSchemaChange(
 		sqlDB.Exec(t, restoreQuery, localFoo)
 		// Wait for all jobs to terminate. Some may fail since we don't restore
 		// adding spans.
-		sqlDB.CheckQueryResultsRetry(t, "SELECT * FROM crdb_internal.jobs WHERE job_type = 'SCHEMA CHANGE' AND NOT (status = 'succeeded' OR status = 'failed')", [][]string{})
+		//
+		// Because crdb_internal.invalid_objects is a virtual table, by default, the
+		// query will take a lease on the database sqlDB is connected to and only run
+		// the query on the given database. The "" prefix prevents this lease
+		// acquisition and allows the query to fetch all descriptors in the cluster.
+		sqlDB.CheckQueryResultsRetry(t, `SELECT * FROM "".crdb_internal.jobs WHERE job_type = 'SCHEMA CHANGE' AND NOT (status = 'succeeded' OR status = 'failed')`, [][]string{})
```

Comment:
nice comment, wrong query 😋 We want the one below

Reply:
😎